lizrd/support/code_versioning.py

```diff
@@ -63,8 +67,8 @@ def version_code(
         ensure_remote_config_exist(repo, REMOTE_NAME, REMOTE_URL)
 
         repo.git.add(all=True)
-        if experiment_config_path is not None:
-            repo.git.add(experiment_config_path, force=True)
+        if files_to_force_add is not None:
+            repo.git.add(files_to_force_add, force=True)
```

Comment:
Sprawdź że na pewno jak masz jakieś zagnieżdżone dziedziczenie configów (tylko takich które nie są by default w repo, np. stwórz dwa poza repo i niech jeden dziedziczy po drugim i na nim odpal) to to się dobrze wrzuca na cemetery. Wydaje mi się że powinno być git, ale lepiej się upewnić

Reply:
Works